### PR TITLE
Andrii's sign for support RMS

### PR DESCRIPTION
**Pull request by andriisgit**
Andrii's sign for support RMS

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -23,7 +23,6 @@ To sign, please [submit a pull request](https://github.com/rms-support-letter/rm
 
 
 Signed,
-[yuriv](https://github.com/yuriv)
```
Comment (nekonee) [inline index.md:26]: Please don't delete other signatures.

File: index.md [hunk 2]
```diff
@@ -217,10 +216,6 @@ Signed,
 
 [Vladimir Zorin](https://github.com/epicfilemcnulty)
 
-[Meri Amirkhanian](https://github.com/meri22)
-
-[George Linkovsky](https://github.com/Timofffee)
-
```
Comment (nekonee) [inline index.md:223]: Please don't delete other signatures.

File: index.md [hunk 3]
```diff
@@ -217,10 +216,6 @@ Signed,
...
 [Aleksej Serdyukov](https://github.com/aleksejrs)
 
-[Serge Postrash](https://github.com/SDKiller)
```
Comment (nekonee) [inline index.md:226]: Please don't delete other signatures.

**Review by nekonee — CHANGES_REQUESTED** [review]

**Comment by aleksejrs** [comment]
That happens when you the original is changed before you send the pull request.

**Comment by podvornyakva** [comment]
Please. add a file instead of edit index.md

Follow README.md instructions.

Thank you.

**Comment by imachug** [comment]
Merged manually.

**Pull request closed without merging**